develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/07-upgrading-application-upgrade-and-verifier-processes.markdown

```diff
@@ -0,0 +1,385 @@
+# Upgrading Application Upgrade and Verifier Processes
+
+Since Liferay 7.0 and Lifery DXP, a new upgrade framework and verifier framework 
+is available for you to use for your application. You can upgrade your existing 
+upgrade and verify processes to use the new framework in just a few steps.
+
+This tutorial demonstrates how to:
+
+- Upgrade an existing upgrade process to the new upgrade framework
+- Upgrade an existing verify process to the new verify framework
+
+Go ahead and get started with the upgrade process next.
+
+## Upgrading Your Upgrade Process to the New Framework
+
+Follow the steps below to migrate your code to the new framework.
+
+1.  If your application has any dependencies, add a dependency on the 
+    `portal-upgrade` module to your `build.gradle` file:
+    
+        provided group: "com.liferay", name: "com.liferay.portal.upgrade", 
+        version: "2.0.0"
```

Comment:
Will there be cases in which the developer's application does not already use a dependency tool, such as Gradle, Maven, or Ivy? For example, might their application be a traditional Plugins SDK app that doesn't use them? If so, this step should speak to that and give them a link to instructions (e.g., [Adapting to Liferay 7's API with the Code Upgrade Tool](/develop/tutorials/-/knowledge_base/7-0/adapting-to-liferay-7s-api-with-the-code-upgrade-tool) ) for handling such a case.